### PR TITLE
Add --name option to kr pair

### PR DESCRIPTION
**Pull request by petrsnm**
I agree to license all rights to my contributions in each modified file exclusively to KryptCo, Inc.

### CODE REVIEW

File: kr/kr.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 	"path/filepath"
 	"strings"
 	"time"
-
+
```
Comment (kcking) [inline kr/kr.go]: nit: extra whitespace. running `gofmt` on each changed file should clean up any formatting issues

File: pair.go [hunk 2]
```diff
@@ -27,6 +27,10 @@ type PairingSecret struct {
 	sync.Mutex
 }
 
+type PairingOptions struct{
+	WorkstationName string `json:"name"`
```
Comment (kcking) [inline pair.go]: For optional JSON fields, I prefer `*string` so that you can always distinguish between `""` and not-present.

File: pair.go [hunk 3]
```diff
@@ -52,13 +56,17 @@ func (ps *PairingSecret) SQSBaseQueueName() string {
 	return strings.ToUpper(derivedUUID.String())
 }
 
-func GeneratePairingSecret() (ps *PairingSecret, err error) {
+func GeneratePairingSecret(workstationName string) (ps *PairingSecret, err error) {
```
Comment (kcking) [inline pair.go]: `string` -> `*string`

File: krd/control_server_test.go [hunk 4]
```diff
@@ -6,9 +6,11 @@ import (
 	"net/http/httptest"
 	"testing"
 	"time"
+	"bytes"
 
 	"github.com/kryptco/kr"
 	"github.com/op/go-logging"
+	"bytes"
```
Comment (kcking) [inline krd/control_server_test.go]: imported twice

File: kr/kr.go [hunk 5]
```diff
@@ -417,6 +424,10 @@ func main() {
 					Name:  "force",
 					Usage: "Do not ask for confirmation to unpair a currently paired device.",
 				},
+				cli.StringFlag{
+					Name:  "name",
```
Comment (kcking) [inline kr/kr.go]: We can make this `"name, n"` so there is a shorthand flag for it as well.

**Comment by petrsnm** [comment]
Requested changes have been made (commit 2 of 2).

**Comment by kcking** [comment]
Thanks!

**Pull request merged**